content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -405,6 +410,113 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry environment
+
+In OpenTelemetry environments, you can use the [OpenTelemetry Collector][16] to collect and batch telemetry data (like metrics, traces, and logs) by leveraging the [Datadog Exporter][17] to send the data to Datadog without the Datadog Agent.
+
+Optionally, you can send telemetry data from [applications instrumented with the OpenTelemetry SDKs with the Datadog Agent][20], which collects the data for Datadog.
+
+To set up unified service tagging in an OpenTelemetry environment:
+
+1. Set the required attributes to ensure proper tagging for the OpenTelemetry component you are using.
+
+   For example, with the Python OpenTelemetry SDK:
+
+   ```python
+   from opentelemetry.sdk.resources import Resource
+   from opentelemetry.sdk.trace import TracerProvider
+
+   resource = Resource(attributes={
+      "service.name": "<SERVICE>",
+      "deployment.environment": "<ENV>",
+      "service.version": "<VERSION>"
+    })
+    tracer_provider = TracerProvider(resource=resource)
+    ```
+
+    For example, with the OpenTelemetry Collector:
+
+    ```yaml
+    receivers:
+    otlp:
+      protocols:
+        grpc:
+        http:
+
+    processors:
+      batch:
+
+    exporters:
+      datadog:
+        api:
+          key: "<DATADOG_API_KEY>"
+        metrics:
+          attributes:
+            service.name: "<SERVICE>"
+            deployment.environment: "<ENV>"
+            service.version: "<VERSION>"
```

Comment:
There is no such `metrics::attributes` key, you would need to use the [transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor) for this. I don't think we need a full Collector configuration here, just the transform processor configuraiton